js/script.js

```diff
@@ -16,8 +20,153 @@ function toggleOpenMenu() {
   }
 }
 
-hamburger.addEventListener('click', toggleOpenMenu);
+function createLanguagesUlElement(languages) {
+  let markup = '<ul>';
+  languages.forEach((language) => {
+    markup += `<li>${language}</li>`;
+  });
+  markup += '</ul>';
+  return markup;
+}
+
+const markupModal = (projectId) => {
+  const project = projects[projectId];
+  return `<div class="project-margin">
+            <div class="project-modal-container">
+              <div class="modal-header">
+                <h2>${project.name}</h2>
+                <img src="./assets/img/close-icon.png" class="close" alt="Close Modal">
+              </div>
+              ${createLanguagesUlElement(project.languages)}
```

Comment:
- [ ] Kindly do the changes that the other reviewer requested you to do. You have to decrease the size of the list of languages following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Keep coding 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/163483764-460e3279-3171-4fe9-a40f-5bb6742acbe7.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/163483796-e0bb6d6f-4471-4d21-906b-a2417df35452.png)

Reply:
Hi @Donard97 

Thank you for your kind feedback. The change has already been effected. Kindly find in the image below:
![image](https://user-images.githubusercontent.com/20554583/163488413-de2714c2-1d21-4d43-bbaa-1f9cb0cdf76c.png)


Sorry, I didn't note that the push was unsuccessful before requesting another review. 

Thanks